_includes/manuals/1.10/4.4.8.3_prov_windows.md

```diff
@@ -0,0 +1,167 @@
+required and have defaults. For the most up to date desciption see the template itself.
+
+##### Important parameters
+1. Required
+
+- `windowsLicenseKey`: Valid Windows license key or generic KMS key
+- `windowsLicenseOwner`: Legal owner of the Windows license key
+- `wimImageName`: WIM image to install from a multi image install.wim file
+
+__Note:__ The correct value for `wimImageName` depends on your _install.wim_. The provisioning _will fail_ an incorrect value is supplied for a multi image WIM file and gets silently ignored if the image contains one image only. 
+
+2. Optional
+The following parameters are only applied if they exist. Some, like `domainJoinAccount` and `domainJoinAccountPasswd` require each other.
+
+- `systemLocale`: en-US
+- `systemUILanguage`: en-US
+- `systemTimeZone`: Pacific Standard Time - see [MS TimeZone Naming](https://msdn.microsoft.com/en-us/library/ms912391%28v=winembedded.11%29.aspx)
+- `localAdminAccountDisabled`: false - will keep the local administrator account disabled (default windows)
+- `ntpSever`: time.windows.com,other.time.server - ntp server to use
+- `domainJoinAccount`: administrator@domain.com - use this account to join the computer to a domain
+- `domainJoinAccountPasswd`: Pa55w@rd - Password for the domain join account
+- `computerOU`: OU=Computers,CN=domain,CN=com - Place the computer account in specified Organizational Unit
+- `computerOuSuffix`: Used if `computerOU` is not present to generate the computer OU from host group and host domain. `computerOU` takes precedence! Note, the OU must still be manually created in active directory
+- `computerDomain`: domain.com - domain to join
+
+##### Troubleshooting
+
+* __Templates__
+
+The templates most likely need a lot of testing to work. This is not covered here; though some hints how to start. You should proceed in this order:
+
+1. __Get your templates to render correctly__. Create random `Bare Metal` host in the desired host group for this purpose and make extensive use of [foreman's template preview feature](https://www.youtube.com/watch?v=KQnaRAI8rf4).
+2. __Continue tesing with VMs__ to test netbooting and basic installation
+3. __Debug `peSetup.cmd`__ by pausing it at the send (remove the comment from `::PAUSE`). Then, use `Ctrl-C` to cancel the script altogether. This way you can debug the rendered `peSetup.cmd` quite nicely in WinPE (eg, `notepad peSetup.cmd`)
+3. The `WAIK Finish` template uses `sDelete.exe` to remove all rendered commands from the provided host. Comment out all `sDelete` commands to debug finish scripts.
+4. Use a manual installed host to test rendered snippets like `WAIK extraFinishCommands` directly.
+5. __Examine `C:\foreman.log.`__ - the output left from the finish script. Also, comment out the clean up stage in the finish script to examine and test the rendered scripts directly.
+
+* __Netbooting__
```

Comment:
I think you should remove the "*" from these headings, this one in particular appears as part of the list above!